package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "format:packages": "pnpm -F './packages/**' format",
     "build:apps": "pnpm -F './apps/**' build:static",
-    "build:packages": "pnpm -F './packages/**' build"
+    "build:packages": "pnpm -F './packages/**' build",
+    "move:apps": "OUTPUT='../../../.workloads' pnpm -F workloads-manager move",
+    "move:apps:desktop": "OUTPUT='~/Desktop/workloads' pnpm -F workloads-manager move"
```

Comment:
The "~" hints to the move function to look up your home directory and replace the "~" with that value.